packages/web/src/js/ScrollView.ts

```diff
@@ -0,0 +1,127 @@
+// The `ScrollView` component is currently untested. This is because it is not possible to test it in a headless
+// environment (tests fail on `getBoundingClientRect` that we need to mock somehow).
+
+import BaseComponent from './BaseComponent';
+import { EventHandler, SelectorEngine } from './dom';
+import { debounce, enableToggleAutoloader } from './utils';
+
+const NAME = 'scrollView';
```

Comment:
To be consistent in naming, I am preferring the Bootstrap style, e. g. use the prefix first and group constant using them. `EVENT_`, `SELECTOR_`, `CLASS_NAME_`...

Reply:
But this is the plugin's name. The other plugins use the same form…

Reply:
I mean the lines down there. I have fixed this.